rules/aip0123/aip0123.go

```diff
@@ -35,7 +35,9 @@ func AddRules(r lint.RuleRegistry) error {
 		resourceAnnotation,
 		resourceNameField,
 		resourcePattern,
+		resourcePlural,
 		resourceReferenceType,
+		resourceSingular,
```

Comment:
I think another idea for these annotations, `singular` and `plural` is to ensure that they appear in the `pattern`(s) as the resource_id segment and collectionId segment respectively.

You can save it for another PR though, feel free to file another issue or add it to the existing issue if you'd like, for future implementation by you or someone else.

Reply:
ack,tracking here: https://github.com/googleapis/api-linter/issues/1146